.github/workflows/docker-build.yml

```diff
@@ -145,10 +165,19 @@ jobs:
     runs-on: ubuntu-latest
     needs: [get-version, setup]
     steps:
-      - name: Check out the code at a specific ref
-        uses: actions/checkout@v4
+      - name: Setup Environment
+        uses: ./.github/actions/setup-uv
         with:
-          ref: ${{ inputs.nightly_tag_main || github.ref }} 
+          ref: ${{ inputs.nightly_tag_main || github.ref }}
+
+      - name: Install the project
+        run: |
+          if [[ "${{ inputs.release_type }}" == "base" ]]; then
+            cd src/backend/base && uv sync --no-dev --no-sources
```

Comment:
was hoping this would fix the issue, but it does not seem to. Still seeing: 
```
 0.467 error: could not find root package `langflow-base-nightly`
 ```